### PR TITLE
Implement take backup

### LINKED ISSUE #303: Implement take backup
Take backup button doesn't do anything.

### PR DESCRIPTION
**Pull request by eatonphil**
Closes #303

### CODE REVIEW

File: test/data.js [hunk 1]
```diff
@@ -133,6 +133,17 @@ export const testLinode = {
...
       },
+      backup_54778596: {
+        type: 'snapshot',
+        id: 'backup_54778593',
+        created: '2016-06-09T15:11:55',
+        finished: '2016-06-09T15:12:55',
+        status: 'successful',
+        datacenter: {
+          label: 'Newark, NJ',
+          id: 'datacenter_6',
```
Comment (na3d) [inline test/data.js]: newark

Comment (eatonphil) [inline test/data.js]: Not yet

Comment (eatonphil) [inline test/data.js]: Wait huh? This was supposed to be implemented already in the API but I took this data from an API response.

**Comment by na3d** [comment]
👍

**Comment by eatonphil** [comment]
thanks!

**Pull request merged**